README.md

```diff
@@ -99,6 +99,30 @@ See the configuration below:
     'middleware' => ['web'],
 ```
 
+## Authentication
+
+You can easily add authentication to log viewing routes using popular `auth` middleware in the `config/log-viewer.php`.
```

Comment:
I think a link to https://laravel.com/docs/9.x/authentication#protecting-routes would be useful here, so that devs can read about it just in case.

Reply:
added